build/main.mk

```diff
@@ -290,7 +291,6 @@ XCSOAR_SOURCES := \
 	$(SRC)/Logger/GlueFlightLogger.cpp \
 	$(SRC)/NMEA/Info.cpp \
 	$(SRC)/NMEA/MoreData.cpp \
-	$(SRC)/NMEA/GPSState.cpp \
```

Comment:
Why?

Reply:
not done on purpose, maybe it was Vagrant User's revenge for trying to get rid of him :-P